packages/cli/src/rollup-plugins/typescript-declarations/create-generator.ts

```diff
@@ -164,7 +165,7 @@ export async function createDeclarationCreator(
               if (
                 !allDeps.has(resolvedModule.resolvedFileName) &&
                 !resolvedModule.isExternalLibraryImport &&
-                resolvedModule.resolvedFileName.includes(dirname)
+                resolvedModule.resolvedFileName.includes(normalizedDirname)
```

Comment:
@RafalFilipek this should basically fix your issue 😃

Reply:
I'll tattoo this line on my back if it's true :D

Reply:
Better start looking for a tattoo artist 😁